drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -113,7 +113,8 @@
 #define SYSCLK_FREQ_MAX		550000000UL
 #define AHB_FREQ_MAX		275000000UL
 #define APBx_FREQ_MAX		137500000UL
-#elif defined(CONFIG_SOC_STM32H7A3XX)
+#elif defined(CONFIG_SOC_STM32H7A3XX) ||\
+	  defined(CONFIG_SOC_STM32H7A3XXQ)
```

Comment:
Only difference between `CONFIG_SOC_STM32H7A3XX` and `CONFIG_SOC_STM32H7A3XX` is SMPS support, so everywhere else, `if defined(CONFIG_SOC_STM32H7A3XX)` should work for both